work/worker.go

```diff
@@ -1015,17 +1037,22 @@ func (env *Task) commitBundleTransaction(bundle *builder.Bundle, bc BlockChain,
 	return nil, nil, logs
 }
 
-func (env *Task) shouldDiscardBundle(bundle *builder.Bundle) bool {
+func (env *Task) shouldDiscardBundle(bundle *builder.Bundle) (bool, error) {
 	if !bundle.TargetRequired {
-		return false
+		return false, nil
 	}
 
 	if env.tcount == 0 {
-		return bundle.TargetTxHash != common.Hash{}
+		return bundle.TargetTxHash != common.Hash{}, errors.New("absent of target tx hash")
```

Comment:
```suggestion
	if env.tcount == 0 {
		return true, fmt.Errorf("target tx %s does not precede the bundle", bundle.TargetTxHash.Hex())
	}
```

Reply:
This suggestion is understood based on https://github.com/kaiachain/kaia/pull/639#discussion_r2536116077. A snippet below seems more readable. PTAL.
```
if env.tcount == 0 && bundle.TargetTxHash != (common.Hash{}) {
```

Reply:
On second thought, the suggestion looks better. @hyeonLewis Do you have some inputs here?

Reply:
Both ways look good, but slightly prefer the suggestion.

Reply:
Fixed at https://github.com/kaiachain/kaia/pull/639/commits/68afe27530cbd43ce3a5336e43612424a1627032

Reply:
The change is identical with the previous one. I guess that you're mentioning to add `env.tcount == 0` to this condition https://github.com/kaiachain/kaia/commit/68afe27530cbd43ce3a5336e43612424a1627032#diff-aa53dfe64c81ad7f4673af6857ca8ae7873bb18224202adb09f765ce4f458411R1045 right? Since the target tx is not specified, `env.tcount` does not appear to need to be looked up.

Reply:
> Since the target tx is not specified

It means we must check `env.tcount == 0` since if not, we should discard bundle.

Sorry for the unclear point; what I preferred was to return `true` or `false` explicitly, following other branches.

Reply:
At the code level, both look the same. I couldn't get your point. Could you suggest a concrete code snippet?